.github/workflows/build_cmake_tarball.yml

```
@@ -34,8 +34,8 @@ on:
     inputs:
       TEST_LEVEL:
         required: true
-        type: number
-        description: 'Test level used for configuring (0 for full tests, 1 for less thorough tests, 2 for minimal tests)'
+        type: string
+        description: 'Test level used for configuring (T8_TEST_LEVEL_FULL, T8_TEST_LEVEL_MEDIUM, or T8_TEST_LEVEL_BASIC)'
```

What about using choice instead? It will resolve into a string
```suggestion
        type: choice
        options:
          - T8_TEST_LEVEL_FULL
          - T8_TEST_LEVEL_MEDIUM
          - T8_TEST_LEVEL_BASIC
        description: 'Test level used for configuring'
```

---

That does indeed sound like the better choice (_pun intended_), I will change it 👍

---

And thanks for the quick review!